packages/client/src/components/sharefiles/uploader.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { compose } from 'recompose';
+import Uppy from '@uppy/core'
+import Tus from '@uppy/tus'
+import { Dashboard } from '@uppy/react'
+import PageTitle from 'components/pageTitle';
+import PageBody from 'components/pageBody';
+import Breadcrumbs from 'components/share/breadcrumb';
+import { withAccessToken, AccessTokenComponentProps } from 'components/helpers/withAccessToken';
+import { withGroupContext, GroupContextComponentProps } from 'context/group';
+import '@uppy/core/dist/style.css';
+import '@uppy/dashboard/dist/style.css';
+
+type Props = AccessTokenComponentProps & GroupContextComponentProps;
+
+const breadcrumbs = [
+  {
+    key: 'browse',
+    matcher: /\/browse/,
+    title: 'Share Files',
+    link: '/browse'
+  }
+];
+
+const ShareFilesUploader = (props: Props) => {
+  const { accessToken, groupContext } = props;
+  const endpoint = 'https://hub.a.demo.primehub.io/api/tus';
```

Comment:
this should be using `config.graphqlHost`

Reply:
wo I forgot update it